### PR TITLE
[Term Entry] PyTorch Tensor Operations .index_add() #5614

### LINKED ISSUE #5614: [Term Entry] PyTorch Tensor Operations .index_add()
### Reason/inspiration (optional)

We would like a new entry on the `.index_add()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensor-operations/terms/index-add/index-add.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by LilyS222**
### Description

New term entry for the tensor operation `.index_add()` in Pytorch.

### Issue Solved

Closes #5614

### Type of Change

- Adding a new entry

### Checklist

- ☑️ All writings are my own.
- ☑️ My entry follows the Codecademy Docs style guide.
- ☑️ My changes generate no new warnings.
- ☑️ I have performed a self-review of my own writing and code.
- ☑️ I have checked my entry and corrected any misspellings.
- ☑️ I have made corresponding changes to the documentation if needed.
- ☑️ I have confirmed my changes are not being pushed from my forked `main` branch.
- ☑️ I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- ☑️ I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensor-operations/terms/index-add/index-add.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Syntax
+
+```pseudo
+torch.index_add(input, dim, index, source)
+```
+
+- `input`: The tensor to which values will be added.
+- `dim`: The dimension along which to index and add.
+- `index`: The indices of elements to add to.
+- `source`: The tensor containing values to add.
```
Comment (SaviDahegaonkar) [inline content/pytorch/concepts/tensor-operations/terms/index-add/index-add.md:27]:
```suggestion
- `input`: The tensor to which values will be added.
- `dim`: The dimension along which to index and add.
- `index`: It is a 1-D tensor with the element indices to be added to.
- `source`: The tensor containing values to add.
- `alpha`: It is a scalar multiplier which is used before adding the source values. It is an optional parameter.
- `out` : If provided, the result will be written to this tensor. It is also an optional parameter.

```

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @LilyS222 ,
Thank you for Contributing to Docs :) I had a look at your file and suggested a few changes. Please make them asap.

Thanks,
Savi

**Comment by SaviDahegaonkar** [comment]
Hey @LilyS222 ,
Thank You for your Contribution :) This entry looks good for a second round of review.

**Review by SaviDahegaonkar — APPROVED** [review]

**Review by Sriparno08 — APPROVED** [review]
Looks good, @LilyS222!

**Pull request merged**